cli/src/opts/forge.rs

```diff
@@ -143,6 +144,9 @@ pub enum Subcommands {
     #[clap(alias = "in", about = "Get specialized information about a smart contract")]
     Inspect(inspect::InspectArgs),
 
+    #[clap(alias = "up", about = "Uploads abi to sig.eth.samczsun.com function selector database")]
```

Comment:
the abi of Contract, right?

should we also output the URL so a user can just click and have a look at the website?

Reply:
assuming you just mean adding `https://` so terminals can hyperlink it? If so, fixed!